packages/website/docs/createRecentSearchesPlugin.md

```diff
@@ -142,3 +204,5 @@ const recentSearchesPlugin = createRecentSearchesPlugin({
 #### `getAlgoliaSearchParams`
 
 > `SearchParameters => SearchParameters`
+
+Optimized [Algolia search parameters](https://www.algolia.com/doc/api-reference/search-api-parameters/). This is useful when using the plugin along with the [Query Suggestions](createQuerySuggestionsPlugin) plugin.
```

Comment:
This was the "best" generic explanation I could come up with. If I missed the core usage, don't hesitate to make a suggestion.

Reply:
```suggestion
Optimized [Algolia search parameters](https://www.algolia.com/doc/api-reference/search-api-parameters/). This is useful when using the plugin along with the [Query Suggestions](createQuerySuggestionsPlugin) plugin.

This function enhances the provided search parameters by:

- Excluding Query Suggestions that are already displayed in recent searches.
- Using a shared `hitsPerPage` value to get a group limit of Query Suggestions and recent searches.
```